login-form/src/components/page-components/Dashboard.js

```diff
@@ -62,11 +68,11 @@ const Dashboard = ({
 const mapStateToProps = (state) => {
   console.log("state", state);
```

Comment:
remove `console.logs()`. As I said multiple times re-check what you are about to commit.